content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+
+The framework he suggests then starts from an ontology made of "mental representations", which correspond to those defined and studied by classical cognitive science, and "public representations", which are the expressions of mental representations in diverse cultural artefacts such as pieces of text, utterances, pictures, myths, built structures, etc..
+New mental representations are constantly formed in people's minds whenever they perceive or interpret public representations.
+For instance, say I am thinking of a tune (mental representation), and I whistle it (public representation);
+someone else hears it, and forms their own mental representation.
+Most of the time, the new representation in that person's head is different from my original representation.
+This last point is a defining feature of the theory Sperber proposes, in contrast to not only memetics, but also to SCE as outlined above [@sperber_explaining_1996, p. 25-26, 31].
+
+On this basis, Sperber proposes to model human societies as large dynamical systems of people continuously interpreting public representations into mental representations, and producing new public representations through their situated actions (in which mental representations play a role).
+To explain culture then, in this framework, is to analyse the processes by which representations circulate through a society, with different levels of change along the way.
+Those processes are many and heterogeneous, which corresponds to the diversity of cultural domains that exist in societies, but the basic ontology remains grounded in the same notion of mental representations from cognitive science.
+
+By developing such an ontology to connect disciplines, Sperber proposes a credible bridge between the notions of representation in social science and that of mental representation in cognitive science, without reducing one area to the other or making simplistic assumptions about the phenomena encountered.
+In this sense, his proposal is that of a naturalistic ontology for the study of culture which builds on cognitive science principles.
+It is amenable to anthropology, and encourages the combination of the two bodies of knowledge in a well defined way.
+Sperber can then rephrase interdisciplinary questions in terms of spread and transformation of representations.
+For instance:
+what types of representations are less transformed than others as people integrate them or perceive them, and produce them anew, making them circulate in a society?
+Such representations, spreading wider than others, become *cultural representations* and will characterise a given society (for instance the habit of clothing, eating customs or values, or technological knowledge).
+
+Why are those representations so stable, and how do they evolve?
+Sperber introduces an additional concept to analyse this evolution:
```

Comment:
TODO
- [ ] explain attractors are a domain of attraction towards which representations tend, by psychological and ecological factors (though it may not be the final cultural representation, see e.g. the smoking example)
- [ ] give examples of attractors: portraits with direct gaze, Little Red Riding Hood, Mornay sauce, anything that's stable through time